build.sbt

```diff
@@ -950,7 +950,8 @@ lazy val liteEmbeddedDeploymentManager = (project in lite("embeddedDeploymentMan
 
   settings(
     name := "nussknacker-lite-embedded-deploymentManager",
-  ).dependsOn(liteEngineKafkaRuntime, deploymentManagerApi % "provided", liteKafkaComponents % "test", testUtil % "test", kafkaTestUtil % "test")
+  ).dependsOn(liteEngineKafkaRuntime, requestResponseRuntime, deploymentManagerApi % "provided", liteKafkaComponents % "test",
```

Comment:
requestResponseRuntime is doubled in 1360

Reply:
:+1: